web/src/common/layout/Layout.svelte

```diff
@@ -13,28 +13,29 @@
 </div>
 
 <style>
-  /* Legacy note: calling this "container" will clash with a pico built-in */
   .flex-container {
     display: flex;
     flex-direction: column;
     height: 100vh;
     margin: 0;
   }
 
-  .top {
-    padding: 4px;
-  }
-
   .content {
     display: flex;
     flex: 1;
     overflow: hidden;
   }
 
   .controls {
-    width: 25%;
-    overflow-y: auto;
-    padding: 10px;
+    width: 25%; /* padding: 10px; */
+    min-width: 350px;
+
+    /* TODO no effect? */
```

Comment:
Some of this seems to have no effect, and also I'm messing up the overflow and nested divs somewhere, because without the overflow here, draw mode can't scroll. Will work on it later.

Reply:
Yes, I made the divs a child of the map-container. You may need to shift that slightly.